x-pack/metricbeat/module/aws/_meta/config.yml

```diff
@@ -4,6 +4,7 @@
     - ec2
   regions:
     - us-east-1
+  calculate_rate: false
```

Comment:
I would always calculate it, the other value is difficult to consume without more context, and adding metrics is non-breaking in principle.

Reply:
Sounds good, I will remove the flag then!